src/bot/commands/user_commands.py

```diff
@@ -125,9 +125,9 @@ def run_get_player_hero_winrate_command(update, context):
         update.message.reply_markdown_v2(constants.USER_NOT_REGISTERED_MESSAGE)
 
     hero_name = " ".join(hero_name_parts)
-    hero = helpers.get_hero_by_name(hero_name)
+    hero_id = hero_services.get_hero_id_by_name_or_alias(hero_name)
```

Comment:
I'm not a huge fan of `find this or that` functions based on input as the internal side effects can get wonky if not done right. I feel like this should be extracted to a helper instead of adding a new function to a service module. A service function should do one thing and on thing only. Determine if its a hero_id by checking the return of the first service, then try again.